Python-packages/covidcast-py/setup.py

```diff
@@ -30,7 +30,8 @@
         "imageio-ffmpeg",
         "imageio",
         "tqdm",
-        "epiweeks"
+        "epiweeks",
+        "importlib_resources==6.1.1",
```

Comment:
Should we be pinning the exact version? I'd assume we just want a compatible version, as in `importlib_resources~=6.1`, or something like that. Or is there a reason we need this exact version?

Reply:
It definitely makes sense for this to less rigid than requiring an exact version, particularly since the other packages are specified with open-ended constraints (or none at all).  Based on the last comment linked gitlab issue, it looks like this could be:
```suggestion
        "importlib_resources>=1.3",
```

Backports are nice for functionality but potentially headache-inducing w.r.t. dependency complexity...  We could make the requirement "`importlib_resources>=1.3;python_version<"3.9"`" but that requires the code to know when it should use `importlib_resources` vs `importlib.resources`